api/src/main/java/io/strimzi/api/kafka/model/AbstractKafkaConnectSpec.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.Affinity;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.fabric8.kubernetes.api.model.Toleration;
+import io.strimzi.api.annotations.DeprecatedProperty;
+import io.strimzi.api.kafka.model.connect.ExternalConfiguration;
+import io.strimzi.api.kafka.model.template.KafkaConnectTemplate;
+import io.strimzi.api.kafka.model.tracing.Tracing;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.KubeLink;
+import io.sundr.builder.annotations.Buildable;
+import io.vertx.core.cli.annotations.DefaultValue;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "replicas", "image",
+        "livenessProbe", "readinessProbe", "jvmOptions",
+        "affinity", "tolerations", "logging", "metrics", "tracing", "template"})
```

Comment:
missing .... version, resources and externalConfiguration

Reply:
I think we should add this check to the `CrdGenerator`, if only to deprive Paolo of these low-hanging fruit review comments.

Reply:
Good idea @tombentley ! Thanks for thinking about me :-)